### ISSUE TITLE
MSVC strange reporting error

### ISSUE THREAD

**Issue by 3038922** [littlevgl/pc_simulator_sdl_visual_studio#2]
![image](https://user-images.githubusercontent.com/33926386/52254440-bdc99680-2948-11e9-8851-c5a05d72cace.png)
I added include like demo
![image](https://user-images.githubusercontent.com/33926386/52254498-03865f00-2949-11e9-8b25-0de3242f1e27.png)
Here is the translation of demo, which can be compiled.
![image](https://user-images.githubusercontent.com/33926386/52254521-20229700-2949-11e9-8ea2-1390d7a61c01.png)

**Comment by embeddedt** [comment]
I don't understand what the problem is. What is the error you're getting? What have you changed from the original project?

**Comment by kisvegabor** [comment]
So the problem is `LV_COLOR_BLACK` (and other colors) is not working?

Can you compile it if without using colors directly?

**Comment by 3038922** [comment]
if i don't use my font ,my img, and set color.
it  can be compiled
But demo uses all the functions and everything works. I suspect it's CPP.

**Comment by 3038922** [comment]
The same code, I use "arm-g++" to compile. It can be compiled and run properly.

**Comment by kisvegabor** [comment]
It seems not related to the library itself once it worked with `arm-g++`.


What do they mean in English?

![image](https://user-images.githubusercontent.com/7599318/52393529-b4564080-2aa6-11e9-9fe9-836e6fa496a2.png)

![image](https://user-images.githubusercontent.com/7599318/52393544-cd5ef180-2aa6-11e9-8902-e94d53cf37c8.png)

**Comment by kisvegabor** [comment]
Have a look at this: https://stackoverflow.com/questions/33270731/error-c4576-in-vs2015-enterprise

**Comment by 3038922** [comment]
> 看看这个：[https](https://stackoverflow.com/questions/33270731/error-c4576-in-vs2015-enterprise)：[//stackoverflow.com/questions/33270731/error-c4576-in-vs2015-enterprise](https://stackoverflow.com/questions/33270731/error-c4576-in-vs2015-enterprise)
> 或尝试手动创建颜色：
> 
> ```c
>     lv_color_t c;
>     c.red = 12 ;
>     c.green = 34 ;
>     c.red = 56 ;
> ```

The parenthesized type followed by the list of initializers is a non-standard explicit type conversion grammar

**Comment by kisvegabor** [comment]
Is it working?
```c
    lv_color_t c;
    c.red = 12 ;
    c.green = 34 ;
    c.red = 56 ;
```

**Comment by 3038922** [comment]
no .
I found that it's the compiler's problem.'arm-none-eabi-g++'Everything's OK.'msvc' seems to have a slightly different grammar.
I wish I had a CMakeLists supporting vscode + MinGW + SDL2. I don't want to use vs.

**Comment by kisvegabor** [comment]
You can try other IDEs too: https://github.com/littlevgl/lvgl#quick-start-in-a-simulator

**Comment by 3038922** [comment]
> You can try other IDEs too: https://github.com/littlevgl/lvgl#quick-start-in-a-simulator

thank you.
The main thing is that we don't install SDL2. The tutorials seem to have problems.

**Comment by kisvegabor** [comment]
The CodeBlock project use native Windows drivers: https://github.com/littlevgl/pc_simulator_win_codeblocks

**Comment by 3038922** [comment]
> The CodeBlock project use native Windows drivers: https://github.com/littlevgl/pc_simulator_win_codeblocks
I'll try vscode. Thank you again for your patience.

**Comment by embeddedt** [comment]
The original issue was fixed by https://github.com/lvgl/lvgl/pull/1711 (will be released with 7.4).
